projects/aca-shared/src/lib/components/toolbar/toolbar.component.scss

```diff
@@ -25,3 +25,7 @@ aca-toolbar {
     margin-right: 5px;
   }
 }
+
+.rotate-icon {
```

Comment:
this is childish... just register the proper image pointing to the expected direction

Reply:
Registered the svg